### PR TITLE
Dantooine: Remove unused files Blazor client

### PR DESCRIPTION
**Pull request by damienbod**

### CODE REVIEW

File: samples/Dantooine/Dantooine.Client/Client/Properties/launchSettings.json [hunk 1]
```diff
@@ -1,29 +0,0 @@
-{
```
Comment (kevinchalet) [inline samples/Dantooine/Dantooine.Client/Client/Properties/launchSettings.json:1]: I'm not so sure about this file as VS keeps recreating it if you try to delete it. Is it really unused?

**Review by kevinchalet — COMMENTED** [review]
Thanks for your PR!

File: samples/Dantooine/Dantooine.Client/Client/Properties/launchSettings.json (hunk 1 above)
Comment (damienbod) [reply samples/Dantooine/Dantooine.Client/Client/Properties/launchSettings.json:1]: yes this is not required, but no idea why it gets created, the server is the only app which is started in the BFF. No idea why this is created or where. I will see if I can find something

**Comment by kevinchalet** [comment]
@damienbod can I merge this PR or do you prefer waiting until we figure out the launchSettings.json mystery? 😃

**Comment by damienbod** [comment]
HI @kevinchalet I think you can merge this, the mystery will take longer and still no idea :) thanks

**Comment by damienbod** [comment]
Wait :) found it <NoDefaultLaunchSettingsFile>true</NoDefaultLaunchSettingsFile> 

Adding this now

**Comment by kevinchalet** [comment]
Nice. Do you confirm there's no impact on the debugging experience? (I'd say no since the server project is the only one that matters, but we never know... 😄)

Please also update the other WASM project so we're consistent.

**Comment by damienbod** [comment]
The other WASM projects require this as far as I understand. The WASM parts in the other projects can be run separately and are secured separately. I started and tested and did not notice any debug problems, saw the debugger in the F12 as well.

**Comment by kevinchalet** [comment]
> The other WASM projects require this as far as I understand. The WASM parts in the other projects can be run separately and are secured separately.

To be clear, I meant the `Balosar.Client` project, which is supposed to be served by `Balosar.Server`. Isn't that project similar to how the Dantooine BFF is served? 😕

**Comment by damienbod** [comment]
The Balosar.Client "can" be hosted in the server, but this is a standalone as well which can be deployed and hosted alone. The SPA WASM can be deployed to  astatic website and has its own OIDC client. The BFF is a view and can only be deployed and run with the server. I think the Balosar.Client should keep this setting which is the default then for an SPA WASM which could be hosted in the server or deployed alone. Balosar is two applications. Dantooine  is one,

**Comment by kevinchalet** [comment]
> The Balosar.Client "can" be hosted in the server, but this is a standalone as well which can be deployed and hosted alone. The SPA WASM can be deployed to a static website and has its own OIDC client.

In theory, I agree, but in practice, one would need to tweak the sample for this scenario to work: CORS is not enabled in `Balosar.Server` and even the OIDC authentication - that sends discovery/JWKS/token requests - won't work if you hosted the 2 apps on different domains.

Let's treat the two samples the same way: either we keep the 2 .json files for both or we remove them for both.

**Comment by damienbod** [comment]
I remove then

**Comment by kevinchalet** [comment]
Merged, thanks for your contribution! 🎉

**Pull request merged**